src/tree/transform.js

```diff
@@ -66,10 +67,10 @@ export const getTranslations = (bBox, height, width) => {
 };
 
 export const applyTransform = (eventTransform, svg, divBox, width, height) => {
-  const scaleFactor = eventTransform.k;
+  const scaleFactor = eventTransform.zoom;
```

Comment:
Could deconstruct here

Reply:
The problem here is that zoom is already defined as an import function

Reply:
ok, could not deconstruct here then 🙃